spec/services/fees/subscription_service_spec.rb

```diff
@@ -52,17 +52,17 @@
   context 'when invoice is on a full period' do
     it 'creates a fee' do
       result = fees_subscription_service.create
-      created_fee = result.fee
 
-      aggregate_failures do
-        expect(created_fee.id).not_to be_nil
-        expect(created_fee.invoice_id).to eq(invoice.id)
-        expect(created_fee.amount_cents).to eq(plan.amount_cents)
-        expect(created_fee.amount_currency).to eq(plan.amount_currency)
-        expect(created_fee.units).to eq(1)
-        expect(created_fee.events_count).to be_nil
-        expect(created_fee.payment_status).to eq('pending')
-      end
+      expect(result.fee).to have_attributes(
+        id: String,
+        invoice_id: invoice.id,
+        amount_cents: plan.amount_cents,
+        amount_currency: plan.amount_currency,
+        units: 1,
+        events_count: nil,
+        payment_status: 'pending',
+        unit_amount_cents: plan.amount_cents
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)